RULES.md

```diff
@@ -136,17 +136,24 @@ There are no warnings for this rule pack.
+| [NIST.800.53-EC2CheckSSHRestricted](https://docs.aws.amazon.com/config/latest/developerguide/restricted-ssh.html)                                       | The Security Group allows unrestricted SSH access - (Control IDs: AC-4, SC-7, SC-7(3)).                         | Not allowing ingress (or remote) traffic from 0.0.0.0/0 or ::/0 to port 22 on your resources helps to restrict remote access.                                                                                                                                                                                                                                 |
+| [NIST.800.53-IAMGroupMembershipCheck](https://docs.aws.amazon.com/config/latest/developerguide/iam-policy-no-statements-with-admin-access)              | The IAM user does not belong to any group(s) - (Control IDs: AC-2(1), AC-2(j), AC-3, AC-6.).                    | AWS Identity and Access Management (IAM) can help you restrict access permissions and authorizations by ensuring IAM users are members of at least one group. Allowing users more privileges than needed to complete a task may violate the principle of least privilege and separation of duties.                                                            |
+| [NIST.800.53-IAMNoUserNoPolicyCheck](https://docs.aws.amazon.com/config/latest/developerguide/iam-user-no-policies-check)                               | The IAM policy is attached at the user level - (AC-2(j), AC-3, AC-5c, AC-6).                                    | Assigning privileges at the group or the role level helps to reduce opportunity for an identity to receive or retain excessive privileges.                                                                                                                                                                                                                    |
+| [NIST.800.53-IAMNoInlinePolicyCheck](https://docs.aws.amazon.com/config/latest/developerguide/iam-no-inline-policy-check)                               | The IAM Group, User, or Role contains an inline policy - (Control ID: AC-6).                                    | AWS recommends to use managed policies instead of inline policies. The managed policies allow reusability, versioning and rolling back, and delegating permissions management.                                                                                                                                                                                |
```

Comment:
```suggestion
| [NIST.800.53-IAMNoInlinePolicy](https://docs.aws.amazon.com/config/latest/developerguide/iam-no-inline-policy-check.html)                               | The IAM Group, User, or Role contains an inline policy - (Control ID: AC-6).                                    | AWS recommends to use managed policies instead of inline policies. The managed policies allow reusability, versioning and rolling back, and delegating permissions management.                                                                                                                                                                                |
```